### DISCUSSION TITLE
当历史的车轮滚到了我的狗头上 | /posts/eval-disappear-2024/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#612] [Announcements]
https://yuanfan.rbind.io/posts/eval-disappear-2024/

**Comment by hongtaoh** [comment]
提到《明朝那些事儿》我就不困了。这套书我 2021 年用了快三个月，每天午饭后一个多小时才一字一句看完。给我的感觉是历史上发生了好多事情，但是现在回头看，感觉又什么都没有发生。这些对自己也是适用的吧，现在再激动人心的事情 1000 年后什么也没有了。

**Reply by earfanfan** [reply]
如果批阅奏折的时候感到困了就去休息呀，哈哈哈，俺的博客就在这里又不会跑掉，哪天想起来再继续来逛逛呗。

《明朝那些事儿》我这次看才看到朱元璋崩了，朱允炆准备削藩。我倒没感觉历史像是什么都没有发生，反而是觉得一切都真实存在过，也越发感受到现在人们的焦虑和痛苦都很真实。

**Reply by hongtaoh** [reply]
>我倒没感觉历史像是什么都没有发生，反而是觉得一切都真实存在过

我的唯心主义碰到了你这个唯物主义者。

>也越发感受到现在人们的焦虑和痛苦都很真实

怎么突然从历史联系到现在？不过其实那时候的痛苦和迷茫和现在并没有什么不同，唯一的不同是现代人的焦虑和痛苦可能更甚。我之前写过一篇草稿，没写完，就借一下你的平台：

>古人就没有欲望吗？有，而且也不少。不过，现代社会的欲望尤其重。我认为一个很重要的原因，我提过不知道多少次了，就是我们了解的太多了。我说“了解的太多”，并不是在提倡愚民政策，让人们什么都不知道。我所说的“了解的太多”，是说知道了太多我们本不用知道的事情，特别是关于别人的事情。

**Reply by earfanfan** [reply]
>我的唯心主义碰到了你这个唯物主义者。

话说我一直没搞明白唯心主义和唯物主义的区别是撒。其实我俩的观点并不是对立的，如果你也觉得世人的焦虑和痛苦都很真实，会帮助你从宏大叙事和虚无中回到真实的生活，而当你遇到一时难以跨越的考验时，这时去想再激动人心的事情百年千年后都不存在了，也会帮助你从情绪漩涡里抽身出来。这样来看，其实观点都是一时的，如果可以帮助自己不被情绪浪潮拍在岸上，都是可以用一下的。

我大概读懂了你写的草稿，你是想说人与人之间的比较会让人的欲望放大，了解了太多别人拥有而自己还未曾拥有的就会很想要也拥有，而不断膨胀的欲望会使内心陷入痛苦之中。这其实是一个千古难题啊，难度等级和我是谁、从哪来到哪去、我活着是为了干嘛差不多。我很多时候也会因为欲望得不到满足而感到很难受，比如看到好书的时候会感慨自己一辈子也达不到那样的水平，或者孤独寂寞的时候也会难过为什么遇不到真正懂得尊重我欣赏我的人，其实这个难题从古到今已经有无数人遇到过，也留下了无数种解法。换个角度看，当你感到痛苦的时候，也许这就是让你认识自己的机会，需要你自己想想该做点撒来放松一下子紧绷的脑子和心。

>怎么突然从历史联系到现在？

大概因为我的脑子并没有限制自己读历史就只能感慨历史吧，就像你看到我写晗晗居然联想到鹿晗一样，咱们脑子里的边界可能比咱们想象得更广阔吧。

**Reply by hongtaoh** [reply]
>或者孤独寂寞的时候也会难过为什么遇不到真正懂得尊重我欣赏我的人

我就很欣赏你啊。在你博客留言的应该都欣赏你吧。

>换个角度看，当你感到痛苦的时候，也许这就是让你认识自己的机会，需要你自己想想该做点撒来放松一下子紧绷的脑子和心。

这是一条好建议。

**Reply by earfanfan** [reply]
我那是举个例子。
